graph_many.py

```diff
@@ -0,0 +1,91 @@
+import re
+import sys
+import collections
+try:
+    import numpy as np 
+except ImportError:
+    print('You need to install numpy.')
+    sys.exit(1)
+try:
+    import plotly.express as px
+except ImportError:
+    print('You need to install plotly.express.')
+    sys.exit(1)
+try:
+    import kaleido
+except ImportError:
+    print('You need to install kaleido.')
+    sys.exit(1)
+
+if len(sys.argv) != 2:
+    print('Usage: %s results.txt' % sys.argv[0])
+    print('Where results.txt is the output of the benchmark script. I.e.')
+    print(' mimalloc-bench/out/bench/benchres.csv')
+    print()
+    print('The script is designed to visualise the results of running the main benchmark')
+    print('script with multiple runs for each benchmark. For example running:')
+    print('  ../../bench.sh -r=10 mi sn allt')
+    print('from the mimalloc-bench/out/bench directory.')
+    print()
+    print('The script will create two graphs, one for time and one for memory.')
+    print('The graphs will be saved as time.html, time.png and time.pdf, and')
+    print('memory.html, memory.png and memory.pdf.')
+    print()
+    print('The graphs will be normalised by the mean time/memory for each benchmark.')
+    print('The normalised time/memory uses a log scale.')
+    sys.exit(1)
+
+r = re.compile('^([^ ]+) +([^ ]+) +([0-9:.]+) +([0-9]+)')
```

Comment:
Please give it a better name <3

Reply:
Done, and applied to your original script.